de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/AModelAccessResourceTest.java

```diff
@@ -196,11 +196,11 @@ public void setUpModel() throws Exception {
 		
 		discipline = RolesFactory.eINSTANCE.createDiscipline();
 		discipline.setName(DISCIPLINE_NAME);
-		discipline.setUser(USER_WITH_REPO_NAME);
+		discipline.getUsers().add(USER_WITH_REPO_NAME);
 		qkNoReference = QudvFactory.eINSTANCE.createSimpleQuantityKind();
 		
 		anotherDiscipline = RolesFactory.eINSTANCE.createDiscipline();
-		anotherDiscipline.setUser("another user");
+		discipline.getUsers().add("another user");
```

Comment:
You're adding the 'another user' to a new discipline - does that make sense?

Reply:
In order to fix it, i replace this code : anotherDiscipline = RolesFactory.eINSTANCE.createDiscipline();
discipline.getUsers().add("another user");
with this: 
anotherDiscipline = RolesFactory.eINSTANCE.createDiscipline();
anotherDiscipline.getUsers().add("another user");